wrangler-proto/src/main/java/io/cdap/wrangler/proto/workspace/v2/Workspace.java

```diff
@@ -159,9 +175,23 @@ public Builder setInsights(JsonObject insights) {
       return this;
     }
 
+    public Builder setNullabilityMap (HashMap<String, UserDefinedAction> nullabilityMap) {
+      this.nullabilityMap = nullabilityMap;
+      return this;
+    }
+
     public Workspace build() {
       return new Workspace(workspaceName, workspaceId, directives, createdTimeMillis, updatedTimeMillis, sampleSpec,
-                           insights);
+                           insights, nullabilityMap);
     }
   }
+
+  /**
+   * UserDefinedAction enum.
+   */
+  public enum UserDefinedAction {
+    SKIP_ROW,
+    SEND_TO_ERROR_COLLECTOR,
+    ERROR_PIPELINE,
```

Comment:
pipeline is not a great name, as wrangler can be used outside of a pipeline when making HTTP requests. This can just be ERROR

Reply:
Done